torchdata/datapipes/iter/transform/flatmap.py

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+from torch.utils.data import DataChunk
+from torch.utils.data import functional_datapipe, IterDataPipe
+
+from typing import TypeVar
+
+T_co = TypeVar("T_co", covariant=True)
+
+
+@functional_datapipe("flatmap")
+class FlatMapDataPipe(IterDataPipe[IterDataPipe[DataChunk[T_co]]]):
```

Comment:
Could you change the name to `FlatMapperIterDataPipe`? We want to name each DataPipe class as `op-er` and functional API as the `op`.